code/datums/weather/weather_types/rain_storm.dm

```diff
@@ -0,0 +1,153 @@
+/datum/weather/rain_storm
+	name = "rain"
+	desc = "Heavy thunderstorms rain down below, drenching anyone caught in it."
+
+	telegraph_message = span_danger("Thunder rumbles far above. You hear droplets drumming against the canopy.")
+	telegraph_overlay = "rain_low"
+	telegraph_duration = 30 SECONDS
+
+	weather_message = span_userdanger("<i>Rain pours down around you!</i>")
+	weather_overlay = "rain_high"
+
+	end_message = span_bolddanger("The downpour gradually slows to a light shower.")
+	end_overlay = "rain_low"
+	end_duration = 30 SECONDS
+
+	weather_duration_lower = 3 MINUTES
+	weather_duration_upper = 5 MINUTES
+
+	weather_color = null
+	thunder_color = null
+
+	area_type = /area
+	target_trait = ZTRAIT_RAINSTORM
+	immunity_type = TRAIT_RAINSTORM_IMMUNE
+	probability = 0
+	turf_weather_chance = 0.01
+	turf_thunder_chance = THUNDER_CHANCE_AVERAGE
+
+	weather_flags = (WEATHER_TURFS | WEATHER_MOBS | WEATHER_THUNDER | WEATHER_BAROMETER | WEATHER_NOTIFICATION)
+
+	/// A weighted list of possible reagents that will rain down from the sky.
+	/// Only one of these will be selected to be used as the reagent
+	var/list/whitelist_weather_reagents = list(/datum/reagent/water)
```

Comment:
this isn't weighted tho

Reply:
This list is used with `pick_weight_recursive()` that accepts regular lists, nested lists, and weighted lists. Other rain subtypes do use weighted lists. (see `/datum/weather/rain_storm/acid`)

Reply:
Tweaked code comments to include it via bf298f94d1dc6953d1ea265f46eb0a09d7889892